bridge/engine/EngineMixer.cpp

```diff
@@ -1165,9 +1165,17 @@ void EngineMixer::checkInboundPacketCounters(const uint64_t timestamp)
         if (!recentActivity && receiveCounters.packets > 5 && inboundContext.activeMedia &&
             inboundContext.rtpMap.format != RtpMap::Format::VP8RTX)
         {
+            inboundContext.activeMedia = false;
+            if (!_engineStreamDirector->streamActiveStateChanged(endpointIdHash, ssrc, false))
+            {
+                logger::warn("streamActiveStateChanged not set. Participant may be removed %zu ssrc %u",
```

Comment:
director's `streamActiveChange` returns false from `setHighestActiveIndex` if there are no changes in state. So log is misleading.